xtuner/_lite/datasets/json.py

```diff
@@ -40,6 +40,7 @@ def __init__(self,
         assert sample_ratio <= 1
         self.tokenize_fn = tokenize_fn
         self.path = path
+        self.tokenizer_workers = int(os.environ.get('TOKENIZER_WORKERS', 8))
```

Comment:
```suggestion
        self.tokenizer_workers = int(os.environ.get('XTUNER_TOKENIZE_WORKERS', 8))
```
TOKENIZER_WORKERS 容易和 HF 的一些环境变量混淆；TOKENIZER 感觉像是 HF TOKENIZER 使用的进程数，最好改成 TOKENIZE